cmd/template/appcatalog/runner.go

```diff
@@ -65,15 +74,16 @@ func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) err
 		if err != nil {
 			return microerror.Mask(err)
 		}
-	}
-	configmapCR, err := appcatalog.NewConfigmapCR(config, configMapData)
-	if err != nil {
-		return microerror.Mask(err)
-	}
 
-	configmapCRYaml, err := yaml.Marshal(configmapCR)
-	if err != nil {
-		return microerror.Maskf(unmashalToMapFailedError, err.Error())
+		catalogConfigMap, err = templateappcatalog.NewConfigMap(config, configMapData)
+		if err != nil {
+			return microerror.Mask(err)
+		}
+
+		catalogConfigMapYaml, err = yaml.Marshal(catalogConfigMap)
+		if err != nil {
+			return microerror.Maskf(unmashalToMapFailedError, err.Error())
+		}
```

Comment:
Diff is a bit hard to read but this was a bug. We should only render the CM if the user set the flag.